README.md

```diff
@@ -949,16 +949,19 @@ googleapis.com domain.
 ### Downscoping with Credential Access Boundaries
 
 [Downscoping with Credential Access Boundaries](https://cloud.google.com/iam/docs/downscoping-short-lived-credentials)
-enables the ability to downscope, or restrict, the Identity and Access Management (IAM) permissions
-that a short-lived credential can use for Cloud Storage.
+enables restricting the Identity and Access Management (IAM) permissions that a
+short-lived credential can use for Cloud Storage. This involves creating a
```

Comment:
Is this intended only for Cloud Storage?

Reply:
From @huangjiahua: Yes, at GA this time Client-Side CAB is for GCS only. We have a verification on server side that only allows GCS resource for the available resources. In the future, AION plans to expand Client-Side CAB to other services.

Reply:
nit: Can you specify that this is something that is currently only supported for GCS and may/ will be expanded to other services in the future?

.i.e.
```
short-lived credential (currently only for Cloud Storage and may be expanded to additional services in the future). This involves creating a
```
or something else

Reply:
Since we're already mentioning that it is used for Cloud Storage, and we don't have a clear timeline of when other services may be supported, I think it's better to keep the language as is.

Reply:
How does your team plan to update the docs when potentially new services are added (say a year or two from now)? I'm just worried that we'll forget to update this when services are added. The wording now makes it seem like this is only supported for GCS. I'd rather it be it sound like GCS (for now) and potentially other services in the future.

Reply:
I'll create a doc to track all the places that need to be updated when the time comes. It includes auth libraries, official docs, and code comments and samples. In the current version of the read me file, we have multiple places that emphasizes on this feature only being available for GCS, for example here:

https://github.com/googleapis/google-auth-library-java/blob/8877624ccde62d999b25ab468bd1d474e36dcd45/README.md?plain=1#L1168-L1169

Reply:
Ok gotcha. Sounds good to me then!